sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_digitaltwins_client.py

```diff
@@ -599,6 +600,9 @@ def delete_event_route(self, event_route_id, **kwargs):
     def query_twins(self, query_expression, **kwargs):
         # type: (str, **Any) -> ~azure.core.async_paging.ItemPaged[Dict[str, object]]
         """Query for digital twins.
+            Note: that there may be a delay between before changes in your instance are reflected in queries.
```

Comment:
> between [](start = 43, length = 8)

Either something missing here or extra word.

Reply:
I think it is OK. (It is copied from the C# comment)